File: PAG/app/Clases/LIBROS_BANCO_DTO.ts



    // $Classes/Enums/Interfaces(filter)[template][separator]
    // filter (optional): Matches the name or full name of the current item. * = match any, wrap in [] to match attributes or prefix with : to match interfaces or base classes.
    // template: The template to repeat for each matched item
    // separator (optional): A separator template that is placed between all templates e.g. $Properties[public $name: $Type][, ]

    // More info: http://frhagn.github.io/Typewriter/

    
    export class LIBROS_BANCO_DTO {
        
        // GESTION
        public GESTION: number = 0;
        // BANCO
        public BANCO: number = 0;
        // CUENTA
        public CUENTA: string = null;
        // MONEDA
        public MONEDA: string = null;
        // TIPO_CUENTA
        public TIPO_CUENTA: string = null;
        // CUENTA_CONTABLE
        public CUENTA_CONTABLE: string = null;
        // SALDO_INICIAL
        public SALDO_INICIAL: number = null;
        // SALDO
        public SALDO: number = 0;
        // INSTITUCION
        public INSTITUCION: number = 0;
        // GA
        public GA: number = 0;
        // UTILIZA_LIBRETAS
        public UTILIZA_LIBRETAS: string = null;
        // SE_CONCILIA
        public SE_CONCILIA: string = null;
        // DEBITA_POR
        public DEBITA_POR: string = null;
        // ACREDITA_POR
        public ACREDITA_POR: string = null;
        // API_ESTADO
        public API_ESTADO: string = null;
        // API_TRANSACCION
        public API_TRANSACCION: string = null;
        // USU_CRE
        public USU_CRE: string = null;
        // FEC_CRE
        public FEC_CRE: Date = null;
        // USU_MOD
        public USU_MOD: string = null;
        // FEC_MOD
        public FEC_MOD: Date = null;
        // DESC_CUENTA
        public DESC_CUENTA: string = null;
    }

File: PAG_DTO/LIBROS_BANCO_DTO.cs
using System;
using System.ComponentModel;

namespace PAG_DTO
{
    public class LIBROS_BANCO_DTO
    {

        [DisplayName("Gestión")]
        public short GESTION { get; set; }
        [DisplayName("Banco")]
        public short BANCO { get; set; }
        [DisplayName("Cuenta")]
        public string CUENTA { get; set; }
        [DisplayName("Moneda")]
        public string MONEDA { get; set; }
        public string TIPO_CUENTA { get; set; }
        public string CUENTA_CONTABLE { get; set; }
        public Nullable<decimal> SALDO_INICIAL { get; set; }
        public decimal SALDO { get; set; }
        public short INSTITUCION { get; set; }
        public short GA { get; set; }
        public string UTILIZA_LIBRETAS { get; set; }
        public string SE_CONCILIA { get; set; }
        public string DEBITA_POR { get; set; }
        public string ACREDITA_POR { get; set; }
        public string API_ESTADO { get; set; }
        public string API_TRANSACCION { get; set; }
        public string USU_CRE { get; set; }
        public System.DateTime FEC_CRE { get; set; }
        public string USU_MOD { get; set; }
        public Nullable<System.DateTime> FEC_MOD { get; set; }

        // Decripciones
        [DisplayName("Decripción")]
        public string DESC_CUENTA { get; set; }

    }
}
